File: Data Structure/Stack/acmicpc.net/9093/Main.java
import java.io.BufferedReader;
import java.io.BufferedWriter;
import java.io.FileInputStream;
import java.io.InputStreamReader;
import java.io.OutputStreamWriter;
import java.util.Stack;
import java.util.StringTokenizer;

class Main{
    static Stack<Character> stack;
    static final String inputFilePath = "./Data Structure/Stack/acmicpc.net/9093/sample.txt";
    public static void main(String[] args) throws Exception{
        System.setIn(new FileInputStream(inputFilePath));
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
        int T = Integer.parseInt(br.readLine());
        stack = new Stack<Character>();        for (int i = 0; i < T; i++) {
            StringTokenizer st = new StringTokenizer(br.readLine());
            while(st.hasMoreTokens()){
                String tmp = st.nextToken();
                for (int j = 0; j < tmp.length(); j++) {
                    stack.add(tmp.charAt(j));
                }
                while(!stack.isEmpty()){
                    bw.write(stack.pop());
                }
                bw.write(" ");
            }
            bw.newLine();;
        }
     bw.flush();   
    }
    
}

File: Data Structure/Stack/acmicpc.net/9093/Main.js
var fs = require("fs");
const inputFilePath = "./Data Structure/Stack/acmicpc.net/9093/sample.txt";
var inputArray;

if (process.platform == "win32") {
    inputArray = fs
        .readFileSync("input.txt", "utf8")
        .toString()
        .trim()
        .split("\n");
} else {
    inputArray = fs
        .readFileSync(inputFilePath)
        // .readFileSync("/dev/stdin")
        .toString()
        .trim()
        .split("\n");
}

class Stack {
    constructor() {
        this._array = [];
    }
    push(item) {
        this._array.push(item);
    }
    pop() {
        return this._array.pop();
    }
}
const T = inputArray[0];
const stack = new Stack();

for (let test_case = 1; test_case <= T; test_case++) {
    const element = inputArray[test_case].split(" ");
    var tmp = "";
    for (let i = 0; i < element.length; i++) {
        for (let j = 0; j < element[i].length; j++) {
            stack.push(element[i].charAt(j));
        }
        for (let j = 0; j < element[i].length; j++) {
            tmp = tmp + stack.pop();
        }
        if (i != element.length - 1) {
            tmp = tmp + " ";
        }
    }
    console.log(tmp);
}
